ivadomed/scripts/training_curve.py

```diff
@@ -145,33 +144,33 @@ def run_plot_training_curves(input_folder, output_folder, multiple_training=Fals
         n_cols, n_rows = 1, 1
 
     for i_subplot, input_folder in enumerate(group_list):
-        input_folder = os.path.expanduser(input_folder)
+        input_folder = Path(input_folder).expanduser()
```

Comment:
That is an interesting feature: https://docs.python.org/3/library/pathlib.html#pathlib.Path.expanduser

Especially interesitn bit: 
![image](https://user-images.githubusercontent.com/6720068/130330002-0bb82224-9838-4df6-9454-4bc12823cad9.png)